### DISCUSSION TITLE
SoapySDR remote questions

### DISCUSSION THREAD

**Discussion by jfath** [rsadsb/dump1090_rs#105] [Q&A]
I'm working on getting dump190_rs and radar running using SoapySDR remote with an Airspy.  It seems to be running, but I'm not actually seeing any airplanes in radar and using netcat to connect to port 30002 shows no output.
Airspy has three gain controls:
    LNA gain range: [0, 15] dB
    MIX gain range: [0, 15] dB
    VGA gain range: [0, 15] dB

My questions:
1)  **Ignore this question** - I see now in the config.toml how each of the gains can be set - sorry
_It seems I can only specify one gain key in the custom-config file unless I'm missing something?_
2) While dump1090_rs is running, it prints a steady stream of characters (mostly 'S', some '0') to stdout.  Is that expected?  Do the characters mean anything?

Thanks for these projects - I was very happy to find an ads-b project that supports remote hw as my sdr and antenna are located in a barn away from the e noise of my house.

**Comment by wcampbell0x2a** [comment]
These are StreamStatusIndicators: https://github.com/pothosware/SoapySDR/wiki/StreamStatusIndicators.

Thanks! I enjoyed writing it.

**Comment by jfath** [comment]
Thanks for the reply.  I'm guessing that this is not normal then.  Probably something specific to remote and/or airspy
S - Sequence error detected. This means that a sequence number was skipped or out of order. Possibly a transmit drop on network-based devices. On transmit, this is detected by the FPGA transmit state machine.

**Comment by wcampbell0x2a** [comment]
Yes, I've only seen `O` when running `dump1090_rs` in debug mode instead of release mode, but if you are doing that it's most likely network related.
